ctrlf.el

```diff
@@ -424,6 +426,14 @@ direction is backwards."
         (previous-history-element 1))
     (ctrlf-previous-match)))
 
+(defun ctrlf--symbol-at-point ()
+  "Return symbol at point.
+When doing regexp search, wrap it with \"\\_<\" and \"\\_>\"."
+  (let ((symbol (thing-at-point 'symbol t)))
```

Comment:
What happens if `thing-at-point` returns nil?

Reply:
Fixed. Now if `thing-at-point` returns nil, the whole function returns nil, so there will be no fake future history element.